actionkit/api/action.py

```diff
@@ -0,0 +1,22 @@
+import json
+import re
+import requests
+
+from actionkit.api import base
+
+class AKActionAPI(base.ActionKitAPI):
+
+    def create_action(self, akid, page, fields=None):
+        data = {
+            'akid': akid,
+            'page': page
+        }
+        if fields is not None:
```

Comment:
might be nice to enumerate valid fieldnames here, even if it's in a comment, but LGTM, as a start.